### DISCUSSION TITLE
Campaign carry over and other ideas

### DISCUSSION THREAD

**Discussion by Yeoldpint** [ihhub/fheroes2#8493] [Ideas]
Suggestion and discussion for Campaign mode adjustment.

**Comment by zenseii** [comment]
Hi, @Yeoldpint. Could you please be more specific about what you have in mind?

**Reply by Yeoldpint** [reply]
Hello
I figured I should try the Fheroes2 first, before making my suggestion...So, I played the expanded campaign Voyage Home. From the scenario 1 to 2, you have the same hero.
How hard would it be to have the campaign hero carry over his experience, levels and skills?

**Reply by oleg-derevenetz** [reply]
Do they carry over in the original game? According to my tests, this is not the case. Why this should be the case in fheroes2 then? The creators of the original campaign did not do this for a reason.

**Comment by Yeoldpint** [comment]
Part of the fun in this game is the music. The original game had some cool opera for the city themes, the expansion changed the all the city themes. Maybe have the city themes on shuffle?
